api/v1alpha1/managedocs_types.go

```diff
@@ -38,9 +38,27 @@ type ManagedOCSSpec struct {
 	ReconcileStrategy ReconcileStrategy `json:"reconcileStrategy,omitempty"`
 }
 
+type ComponentState string
+
+const (
+	ComponentReady    ComponentState = "Ready"
+	ComponentPending  ComponentState = "Pending"
+	ComponentNotFound ComponentState = "Not Found"
```

Comment:
I think somehow the following change got reverted:

```suggestion
	ComponentNotFound ComponentState = "NotFound"
```